Dockerfile

```diff
@@ -15,7 +15,10 @@ RUN bundle install --jobs 2 --retry 3 --no-cache ${BUNDLE_FLAGS}
 
 COPY . .
 
-ADD https://s3.amazonaws.com/rds-downloads/rds-combined-ca-bundle.pem ./rds-combined-ca-bundle.pem
+ADD https://s3.amazonaws.com/rds-downloads/rds-ca-2019-root.pem ./rds-ca-2019-root.pem
+ADD https://s3.amazonaws.com/rds-downloads/rds-ca-2015-root.pem ./rds-ca-2015-root.pem
+RUN cat ./rds-ca-2019-root.pem > ./rds-ca-bundle-root.crt
+RUN cat ./rds-ca-2015-root.pem >> ./rds-ca-bundle-root.crt
```

Comment:
Looks good.

Am I right in saying that we won't be able to boot this container without an internet connection?

Reply:
this happens at build time so it shouldn't affect anything at boot time

Reply:
@emileswarts yes but the first error would be we can't download the ruby:2.6.5-alpine3.9 base image